README.md

```diff
@@ -276,6 +304,10 @@ Compose.prototype.map = function(f) {
 
 #### `sequence` method
 
+```hs
+sequence :: Apply f, Traversable t => t (f a) ~> (b -> f b) -> f (t a)
```

Comment:
I can a point that in laws we use `Applicative` (e.g. "for any Applicative `F`"), although maybe we should use `Apply` in laws :)